test/check-application

```diff
@@ -1184,6 +1196,95 @@ class TestApplication(testlib.MachineCase):
               .selectImageAndDownload() \
               .expectDownloadErrorForNonExistingTag()
 
+    def testPullImage(self):
+        b = self.browser
+        m = self.machine
+
+        self.setupRegistry()
+
+        def buildImage(auth, content, image_name):
+            temp_dir = self.machine.execute(f"""
+                set -eu
+                D=$(mktemp -d -p {self.vm_tmpdir})
+                # Allow admin to read the created directory, mktemp by default creates it unreadable for other users
+                chmod 755 $D
+                echo {content} > $D/test.txt
+                printf 'FROM scratch\\nCOPY test.txt /\\n' > $D/Containerfile
+                echo $D
+            """).strip()
+            self.execute(auth, f"podman build -t {image_name} {temp_dir}")
+
+        image1 = "localhost:5000/test-image:1"
+
+        # build update and push it
+        buildImage(True, "hello world", image1)
+        image1_update_sha = self.execute(True, f"podman inspect --format '{{{{.Id}}}}' {image1}").strip()
+        self.execute(True, f"podman push {image1}; podman rmi {image1}")
+
+        # build image which we try to pull
+        buildImage(True, "hello", image1)
+        image1_sha = self.execute(True, f"podman inspect --format '{{{{.Id}}}}' {image1}").strip()
+
+        self.login()
+
+        image1_sel = f"#containers-images tbody tr[data-row-id=\"{image1_sha}true\"]".lower()
+
+        # Wait new image sha now visible
+        b.click("#containers-images button.pf-v5-c-expandable-section__toggle")
+        b.click(f'{image1_sel} .pf-v5-c-menu-toggle')
+        b.click(image1_sel + " button.pf-v5-c-menu__item:contains('Pull')")
+        b.wait_in_text("div.download-in-progress", "Pulling")
```

Comment:
Pulling a tiny image from a local registry just takes a blink of an eye.. that feels like inviting a flake? (This isn't what the current failures were about, though)